Gemfile

```diff
@@ -132,7 +132,7 @@ end
 
 group :vmware, :manageiq_default do
   manageiq_plugin "manageiq-providers-vmware"
-  gem "vmware_web_service",             "~>0.1.4"
+  gem "vmware_web_service",             "~>0.2.0"
```

Comment:
Does this need to even be here anymore?  Or is there something in platform still referringto the vmware_web_service gem.

Reply:
@agrare how did that happen.  Fixing.

Reply:
@jerryk55 not sure what you mean, what are you fixing?

Reply:
oh - not fixing I misunderstood your comment.  I realize you meant there is no need for my PR bumping manageiq-smartstate since this one does it.  But I need it done for a BZ.  That has nothing to do with vmware.